tests/test_pytest_cpp.py

```diff
@@ -91,6 +92,29 @@ def test_success(facade, name, test_id, exes):
     assert facade.run_test(exes.get(name), test_id)[0] is None
 
 
+def test_cmdline_builder_happy_flow():
+    arg_string = make_cmdline( ["wine"], "gtest", ["--help"] )
+    assert arg_string
+    assert arg_string[0] == 'wine', "First element MUST be a `harness` value"
```

Comment:
Better we check the whole list:

```suggestion
    assert arg_string == ["wine", "gtest", "--help"]
```

Same for the other tests.